source/docs/advanced/theme/layouts.md

```diff
@@ -0,0 +1,130 @@
+├── contact.js
+├── cart.js
+├── ...
+├── _layout.js
+└── blog
+    ├── index.js
+    └── [slug].js
+```
+
+As is, the urls `/blog` and `/blog/[slug]` you have created will be displayed with the same layout as your cart, contact, or any other page of the application.
+
+However you don't the users to feel like they are still in the shop. You want to have a lighter design for these pages to allow the users to focus on the content. This is why you can create a new `_layout.js` at `/blog/_layout.js`.
+
+```jsx
+// web/theme/routes/blog/_layout.js
+import React from "react";
+
+const BlogLayout = ({children}) => (
+  <div>
+    <header>Welcome to our blog!</header>
+    <main>
+      {children}
+    </main>
+  </div>
+);
+
+export default BlogLayout;
+```
+
+The children property here is the content of the route that will be displayed if you navigate to a page within `/blog`.
+
+Other pages will keep the root `_layout.js` and won't be affected by the blog's layout.
+
+To see this live, remember to restart your application before seeing the changes (`npm run start`).
+
+<blockquote class="note">
+    **Important:** If you need to to override an existing `_layout.js` you can by recreating theme in your own module. You can also create layouts for a set of routes even if the routes you want to change are in an other module and don't have a layout yet. It will behave just like if you added them in the other module.
+</bloquote>
+
+<blockquote class="note">
+    **Note:** In Front-Commerce's core, this is already the case for the `/checkout` that uses a leaner layout than the rest of the shop.
+</bloquote>
+
+### Define an Inner Layout for a list of route
+
+The concept of creating an Inner Layout is very similar to creating a Layout. The main difference is that instead of creating a `_layout.js` file, you will create a `_inner-layout.js`.
+
+For instance, in the case of the account, we've got an Inner Layout similar to this file:
+
+```jsx
+// web/theme/routes/account/_inner-layout.js
+import React from "react";
+import Link from "theme/components/atoms/Typography/Link";
+
+const AccountLayout = ({children}) => (
+  <div>
+    <nav>
+      <ul>
+        <li><Link to="/">Dashboard</Link></li>
+        <li><Link to="/orders">Orders</Link></li>
+        <li><Link to="/address-book">Address Book</Link></li>
+      </ul>
+    </nav>
+    <div>
+      {children}
+    </div>
+  </div>
+);
+
+export default AccountLayout;
+```
+
+You can see that it works just like the Blog's layout. However, keep in mind that you don't need to repeat the surrounding layout. For instance, there's no header and logo in my Inner Layout.
+
+To see this live, remember to restart your application before seeing the changes (`npm run start`).
+
+<blockquote class="warning">
+    **Warning:** You can't create an `_inner-layout.js` if a `_layout.js` file already exists at the same level. This is the case for the files in *your* modules but also for files in other's modules.
+</bloquote>
```

Comment:
It may be relevant to include a note about the possibility to create an infinite number of inner layouts that would be nested.